includes/class-wc-siftscience-format-account.php

```diff
@@ -31,7 +31,7 @@ public function create_account( $user_id, WP_User $user ) {
 				// Supported Fields
 				'$session_id'       => $this->options->get_session_id(),
 				'$user_email'       => $user->user_email,
-				'$name'             => $user->first_name . ' ' . $user->last_name,
+				'$name'             => $user->get_formatted_billing_full_name();
```

Comment:
Actually it's neither. In this situation, we're catching the "create user" event, which is a WordPress users and not yet a WooCommerce user.

Reply:
ok